### PR TITLE
#15: gmbal 4.0.1 is missing OSGi headers

### LINKED ISSUE #15: gmbal 4.0.1 is missing OSGi headers
caused by 11ae64b

### PR DESCRIPTION
**Pull request by lukasj**
Fixes #15

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -162,6 +162,10 @@
                         <Private-Package>
                             org.glassfish.gmbal.util, org.glassfish.gmbal.impl, org.glassfish.gmbal.typelib
                         </Private-Package>
+                        <Import-Package>
+                            org.glassfish.pfl.*;version="[\$(version;=-;\$(@)),\$(version;+;\$(@)))",
```
Comment (lukasj) [inline pom.xml]: The version of pfl jumped to 4.1, so by default, [4.1,5) is being generated but what we want is to have [4.0, 5) being imported - assuming that there is no API change between PFL 4.0.1 and 4.1.0. If there is then this should be removed

Comment (russgold) [reply pom.xml]: There were a number of API changes. Unused methods were removed, which should not affect anything, but in addition, some methods were added to the Wrapper and Bridge classes to support JDK11.

Comment (lukasj) [reply pom.xml]: OK, removed this part

**Comment by lukasj** [comment]
this was super-seeded and fixed in #18

**Pull request closed without merging**